build/kobo.mk

```diff
@@ -104,6 +105,7 @@ $(TARGET_OUTPUT_DIR)/KoboRoot.tgz: $(XCSOAR_BIN) \
 	$(Q)install -m 0644 $(topdir)/kobo/inittab $(@D)/KoboRoot/etc
 	$(Q)install -m 0644 $(topdir)/kobo/udev.rules $(@D)/KoboRoot/etc/udev/rules.d/99-xcsoar.rules
 	$(Q)install -m 0644 $(BITSTREAM_VERA_FILES) $(@D)/KoboRoot/opt/xcsoar/share/fonts
+	$(Q)install -T -m 0644 $(CA_FILE) $(@D)/KoboRoot/etc/ssl/certs/ca-certificates.crt
```

Comment:
This will split obscure error messages when there's no CA file in the download directory.

Reply:
> This looks... fragile.

removed.

> This will split obscure error messages when there's no CA file in the download directory.

Changed .tgz file to be dependent upon file in d/l directory.

> Why don't you use the same thing as the Boost download does:

Made like Boost.  Note this will require kobo.mk to be updated from time to time to get the latest CA version (as with boost and many other files).

Reply:
Changed .tgz file to be dependent upon file in d/l directory.